django_netjsonconfig/base/template.py

```diff
@@ -110,12 +121,13 @@ def clean(self, *args, **kwargs):
             self.config = self.vpn.auto_client(auto_cert=self.auto_cert)
 
     def get_context(self):
-        c = {
+        context = {
             'id': str(self.id),
             'name': self.name,
         }
-        c.update(super().get_context())
-        return c
+        context.update(super().get_context())
+        context.update(self.default_values)
```

Comment:
Added the changes in this function, I am fairly certain these changes were lost during rebasing (as it was not done properly done! )

Reply:
The context was initially added with these defaults value just for consistency as the `id and name` does no have any significant impact. @nemesisdesign has asked it should be removed since context can now be taken from `default_values`

Reply:
No, I meant mention of `self.default_values` in this function was missing, 
can you please share how did your initial changes look like? :smile:

Reply:
@atb00ker here is how my initial changes looked like
```
        c = self.default_values or {}
        c.update(super().get_context())
        return c
```

Reply:
Got it, thanks for the information about the change, i'll revert it! :smile: